lib/vsc/mympirun/mpi/mpi.py

```diff
@@ -932,38 +732,38 @@ def make_mpiexec_hydra_options(self):
                 self.mpiexec_options.append("-%s %s" % (self.HYDRA_LAUNCHER_NAME, launcher[0]))
 
             if launcher_exec is not None:
-                self.log.debug("make_mpiexec_hydra_options: HYDRA using launcher exec %s" % launcher_exec)
+                self.log.debug("make_mpiexec_hydra_options: HYDRA using launcher exec %s", launcher_exec)
                 self.mpiexec_options.append("-%s-exec %s" % (self.HYDRA_LAUNCHER_NAME, launcher_exec))
 
     def get_hydra_info(self):
-        """Get a dict with hydra info"""
+        """Get a dict with hydra info."""
```

Comment:
i find this very odd that this data came from `-info` but now works with `-help`. please double check